src/main/java/com/microsoft/spring/data/gremlin/conversion/GremlinSourceEdge.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.spring.data.gremlin.conversion;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class GremlinSourceEdge extends BasicGremlinSource<String> {
+
+    @Getter
+    @Setter
+    private String vertexIdFrom;
+
+    @Getter
+    @Setter
+    private String vertexIdTo;
+
+    public GremlinSourceEdge() {
+        super();
+
+        this.vertexIdFrom = null;
+        this.vertexIdTo = null;
```

Comment:
This initialization is redudant? by default, String variable has null value?

Reply:
Got it, will refine that.